randovania/games/prime1/game_data.py

```diff
@@ -95,6 +95,12 @@ def _exporter():
             "break fixes present on that version are undone. This reverts the game to be functionally equivalent to "
             "NTSC 0-00, meaning that all versions of Prime are guaranteed to be logically completable when randomized.",
         ),
+        (
+            "Why are some rooms in Magmoor Caverns not superheated?",
+            "Elevator rooms, Storage Cavern, and every room between Twin Fires and the Phazon Mines and "
+            "Phendrana Drifts elevators (Far Magmoor) are not superheated, not including Burning Trail "
+            "and Lake Access which can be encountered without Varia Suit in a vanilla playthrough.",
+        ),
```

Comment:
You phrased it as a why then didn't say why.

If keeping it this way, you may want to say `In the vanilla game, only certain areas of magmoor are superheated. The far side beyond Twin Fires Tunnel as well as Storage Caverns are exceptions which can be traversed without heat protection.`
I don't know if it's worth mentioning burning trial etc, unless that last sentence makes sense only with the missing reason.

If you rephrase to `What rooms in Magmoor are superheated?`, you could just list the unheated rooms similar to above.

Reply:
I mainly phrased the question that way since I imagine that would be the general question players would ask. I'll rephrase the answer to work with it better.

Reply:
bump?